packages/components/src/design-token-preview-tables/ColorTokensTable.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from "@testing-library/react";
```

Comment:
Does it make sense to write tests for docs?

Reply:
I think so? At least for these ones since they are being 'rendered' as jsx components.

Reply:
I'm not sure because they are not used in the app. They are components used only on docs they won't break anything on the apps. IMO it looks like testing the tests or components we build to use on stories.